earth_enterprise/src/common/performancelogger.h

```diff
@@ -55,16 +79,45 @@ class BlockPerformanceLogger {
       size(size),
       startTime(getime::getMonotonicTime()),
       ended(false) {}
+    ~BlockPerformanceLogger() { this->end(); }
```

Comment:
Style: I think `end();` reads fine.  There isn't anything requiring an explicit `this`.

Reply:
Will fix, was just always a preference of mine.